packages/larva-patterns/modules/mega-menu-footer/mega-menu-footer.twig

```diff
@@ -1,8 +1,11 @@
 <footer class="mega-menu__footer // {{ mega_menu_footer_classes }} ">
 	<div class="mega-menu__footer-inner // {{ mega_menu_footer_inner_classes }} ">
-		<section class="mega-menu__footer-subscribe // {{ mega_menu_footer_subscribe_classes }} ">
-			{% include "@larva/modules/mega-menu-footer-subscribe/mega-menu-footer-subscribe.twig" with mega_menu_subscribe %}
-		</section>
+
+		{% if mega_menu_subscribe %}
```

Comment:
This if check may cause regressions on other brands

Reply:
Checked with RS and looks good to me, Had to introduce this check as not all brands have subscribe card in mega menu.
SPY does not have the card hence, Introduced the condition.